Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.11.1
+
+WORKDIR /cinema
+
+COPY requirements.txt .
```

Comment:
It's better to specify the source and destination explicitly in the COPY command, like `COPY requirements.txt /app/requirements.txt`, to avoid any ambiguity.